tests/model_fields_/models.py

```diff
@@ -35,11 +35,12 @@ class DecimalParent(models.Model):
 
 class EmbeddedModelFieldModel(models.Model):
     simple = EmbeddedModelField("EmbeddedModel", null=True, blank=True)
-    decimal_parent = EmbeddedModelField(DecimalParent, null=True, blank=True)
+    decimal_parent = EmbeddedModelField(DecimalKey, null=True, blank=True)
```

Comment:
Why the change?

Reply:
Foreign key aren't supported in embedded fields, So I decided to change this fields. (Decimal parent has a FK to another table)